src/igvfd/audit/file.py

```diff
@@ -38,8 +38,11 @@ def audit_upload_status(value, system):
         detail = (
             f'{object_type} {audit_link(path_to_text(value["@id"]), value["@id"])} has `upload_status` {upload_status}.'
         )
+        if upload_status == 'invalidated':
+            validation_error_detail = value.get('validation_error_detail')
+            detail = f'{detail} Validation error detail: {validation_error_detail}'
         yield AuditFailure(
-            'unvalidated upload status',
+            'upload status not validated',
```

Comment:
I have modified the category name because I don't like "unvalidated", this technically breaks our naming conventions for audits but I'm willing to gracefully make an exception.